rsync-downloaded-files.sh

```diff
@@ -1,21 +1,41 @@
 #!/bin/bash
 
+set -e
+
 # Iterate over all the collections that are fully loaded into the database
 
 for collection in $(psql -U ocdskfpreadonly -d ocdskingfisherprocess -h localhost -c "select source_id, to_char(data_version, 'YYYYMMDD_HH24MISS') from collection where store_end_at is not null and transform_type is null" -qAtX | tr "|" "/"); do
 
+        echo "COLLECTION ${collection}"
```

Comment:
This script was originally attended to be run via cron. Probably best to keep it silent if there's no tty, unless this was intentional

Reply:
There is already some output, so I thought we might as well go for some more. We can look at making sure it's logged somewhere nice - that's probably the right thing to do for debugging purposes anyway!